websites/S/SDVX Index/metadata.json

```
@@ -0,0 +1,30 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "189690228292845568",
+		"name": "angelolz"
+	},
+	"service": "SDVX Index",
+	"description": {
+		"en": "A chart viewer for Sound Voltex, providing top-down overviews of charts for players to study and practice."
+	},
+	"url": "sdvxindex.com",
```

There are new fields called `matches` and `iFrameMatches`, which are required for new Presences.
These are an array of strings which need to follow https://developer.chrome.com/docs/extensions/develop/concepts/match-patterns. 

```suggestion
	"url": "sdvxindex.com",
	"matches": [
		"*://www.sdvxindex.com/*",
		"*://sdvxindex.com/*"
	],
```